azuredeploy.json

```diff
@@ -72,6 +72,17 @@
             },
             "type": "bool"
         },
+        "httpsTermination": {
+            "allowedValues": [
+                "VMSS",
+                "None"
+            ],
+            "defaultValue": "VMSS",
+            "metadata": {
+                "description": "Indicates where https termination occurs. 'VMSS' is for https termination at the VMSS instance VMs (using nginx https proxy). 'None' is for testing only with no https. 'None' may not be used with a separately configured https termination layer. If you want to use the 'None' option with your separately configured https termination layer, you'll need to update your Moodle config.php manually for $cfg->wwwroot and $cfg->sslproxy."
```

Comment:
I assume this status is temporary, that is our goal is to automatically configure `config.php`. It might be worth adding "At this time" and perhaps a link to #31 for our longer term plan.

No need to block merging on this though.